.github/workflows/scheduled-check.yaml

```diff
@@ -0,0 +1,277 @@
+name: "Scheduled checks" # Note that this name appears in the README's badge
+on:
+  push:
+  # TEMPORARILY DISABLED
+  # schedule:
+  #   - cron: "32 2,14 * * *"  # 02:32 UTC and 14:32 every day
+
+env:
+  # TODO: flip this to the production DSN once we're done testing
+  SENTRY_DSN: https://c7764face5354517befaee2e56b63cfb@sentry.prod.mozaws.net/122
+  SLACK_NOTIFICATION_WEBHOOK_URL: ${{ secrets.SLACK_NOTIFICATION_WEBHOOK_URL }}
+
+jobs:
+  run_on_cdn_1_4:
+    runs-on: ubuntu-latest
+    name: Check main CDN - first quarter
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Install requirements
+        run: pip install -r requirements.txt
+      - name: Run checker
+        run: >
+          python bin/run_checks.py
+          --sitemap-url=https://www.mozilla.org/sitemap.xml
+          --batch="1:4"
+      - name: Upload output files
+        uses: actions/upload-artifact@v2
+        with:
+          name: scan-results
+          path: output
+
+  run_on_cdn_2_4:
+    runs-on: ubuntu-latest
+    name: Check main CDN - second quarter
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Install requirements
+        run: pip install -r requirements.txt
+      - name: Run checker
+        run: >
+          python bin/run_checks.py
+          --sitemap-url=https://www.mozilla.org/sitemap.xml
+          --batch="2:4"
+      - name: Upload output files
+        uses: actions/upload-artifact@v2
+        with:
+          name: scan-results
+          path: output
+
+  run_on_cdn_3_4:
+    runs-on: ubuntu-latest
+    name: Check main CDN - third quarter
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Install requirements
+        run: pip install -r requirements.txt
+      - name: Run checker
+        run: >
+          python bin/run_checks.py
+          --sitemap-url=https://www.mozilla.org/sitemap.xml
+          --batch="3:4"
+      - name: Upload output files
+        uses: actions/upload-artifact@v2
+        with:
+          name: scan-results
+          path: output
+
+  run_on_cdn_4_4:
+    runs-on: ubuntu-latest
+    name: Check main CDN - fourth/final quarter
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Install requirements
+        run: pip install -r requirements.txt
+      - name: Run checker
+        run: >
+          python bin/run_checks.py
+          --sitemap-url=https://www.mozilla.org/sitemap.xml
+          --batch="4:4"
+      - name: Upload output files
+        uses: actions/upload-artifact@v2
+        with:
+          name: scan-results
+          path: output
```

Comment:
You can likely use the [matrix context](https://docs.github.com/en/actions/learn-github-actions/contexts#matrix-context) to avoid the duplication here.

Reply:
Yeah, good shout - it's on my TODOlist to DRY it up and the matrix approach could be ideal. Thanks!

- [ ] @stevejalim Enter the matrix